code/game/machinery/computer/arcade.dm

```diff
@@ -31,6 +31,7 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		/obj/item/toy/talking/griffin = 2,
 		/obj/item/coin/antagtoken = 2,
 		/obj/item/stack/tile/fakespace/loaded = 2,
+		/obj/item/stack/tile/fakeopenspace/loaded = 2,
```

Comment:
might be neat if this only populated on multi-z maps, on normal maps it would be the same as fakespace right?

Reply:
yeap, but you can use it on lavaland which makes it unique

Reply:
is lavaland multi-z? or does it show the lava underneath, and if so, does it still have the grey vignette effect?
might look kinda odd but those are kinda nitpicks

Reply:
it shows the lava, without vignette